tox.ini

```diff
@@ -102,6 +103,7 @@ deps =
   ; proto 3 and 4 tests install the respective version of protobuf
   proto3: protobuf~=3.19.0
   proto4: protobuf~=4.0
+  proto5: protobuf~=5.27.0
```

Comment:
The comment on line 103 is now somewhat outdated. Maybe change it to

```
  ; proto 3, 4 and 5 tests install the respective version of protobuf
```
like you do below?